babel/src/main/codegen/config.fmpp

```diff
@@ -1001,6 +1002,10 @@ data: {
       "DateTimeTerm()"
     ]
 
+    qualifyClauseMethod: [
```

Comment:
nit: Should there be an 's' at the end of this? Not sure if we'll have multiple <QUALIFY> statements but even if not adding an 's' keeps with the convention.